x-pack/plugins/triggers_actions_ui/public/application/sections/rules_list/components/rules_list.tsx

```diff
@@ -286,6 +329,30 @@ export const RulesList: React.FunctionComponent = () => {
     })();
   }, []);
 
+  useEffect(() => {
+    if (setStatusFilter) {
+      setStatusFilter(ruleStatusesFilter);
+    }
+  }, [ruleStatusesFilter]);
+
+  useEffect(() => {
+    if (statusFilter) {
+      setRuleStatusesFilter(statusFilter);
+    }
+  }, [statusFilter]);
+
+  useEffect(() => {
+    if (lastResponseFilter) {
+      setRuleExecutionStatusesFilter(lastResponseFilter);
+    }
+  }, [lastResponseFilter]);
```

Comment:
nitpick: I think can we make this a bit more concise if we defaulted `statusFilter` and `lastResponseFilter` to empty arrays where we destructure the props. Then we don't have to check for existence here. 

But yes I now see what you mean by trying to sync the props with the local state

Reply:
I applied what you suggested and look what happens. In the end of the video I show you a git diff of the changes I applied, in case I misunderstood your suggestion.

https://user-images.githubusercontent.com/2852703/176601207-8183c895-8869-4906-8067-372bfea1e93c.mov